var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -22,12 +22,8 @@ class PyMorphio(PythonPackage):
     version('2.1.2', tag='v2.1.2', submodules=True, get_full_repo=True)
     version('2.0.8', tag='v2.0.8', submodules=True, get_full_repo=True)
 
-    variant('mpi', default=False, description="Use parallel HDF5 when needed")
-
     depends_on('py-setuptools', type='build')
     depends_on('py-setuptools-scm', type='build')
 
     depends_on('cmake@3.2:', type='build')
     depends_on('py-numpy', type='run')
-    depends_on('hdf5 +mpi', type=('build', 'run'), when='+mpi')
-    depends_on('hdf5 ~mpi', type=('build', 'run'), when='~mpi')
```

Comment:
@GianlucaFicarelli : doesn't py-morphio required hdf5 dependency?

Reply:
Yes, that should stay in here as just `depends_on('hdf5')`. `highfive` comes as a submodule, @pramodk .

Reply:
Yes, I wanted to rollback https://github.com/BlueBrain/spack/pull/1081 but I should leave the `hdf5` dependency without specifying `mpi`.

Reply:
yes! if you fix this, we should be good to go.